external/fv3kube/fv3kube/base_yamls/v0.7/fv3config.yml

```
@@ -0,0 +1,334 @@
+    make_nh: false
+    mountain: true
+    n_split: 6
+    n_sponge: 4
+    na_init: 0
+    ncep_ic: false
+    nggps_ic: false
+    no_dycore: false
+    nord: 2
+    npx: 49
+    npy: 49
+    npz: 79
+    external_eta: true
+    ntiles: 6
+    nudge: false
+    nudge_qv: true
+    nwat: 6
+    p_fac: 0.1
+    phys_hydrostatic: false
+    print_freq: 3
+    range_warn: true
+    reset_eta: false
+    rf_cutoff: 800.0
+    rf_fast: false
+    tau: 5.0
+    use_hydro_pressure: false
+    vtdm4: 0.06
+    warm_start: true
+    z_tracer: true
+  fv_grid_nml: {}
+  fv_nwp_nudge_nml:
+    add_bg_wind: false
+    do_ps_bias: false
+    ibtrack: true
+    k_breed: 10
+    kbot_winds: 0
+    mask_fac: 0.2
+    nf_ps: 3
+    nf_t: 3
+    nudge_debug: true
+    nudge_hght: false
+    nudge_ps: false
+    nudge_virt: false
+    nudge_winds: false
+    nudge_q: false
+    tau_ps: 21600.0
+    tau_virt: 21600.0
+    tau_winds: 21600.0
+    tau_q: 21600.0
+    r_hi: 5.0
+    r_lo: 3.0
+    r_min: 225000.0
+    t_is_tv: false
+    tc_mask: true
+    time_varying: false
+    track_file_name: No_File_specified
+    use_high_top: true
+  gfdl_cloud_microphysics_nml:
+    c_cracw: 0.8
+    c_paut: 0.5
+    c_pgacs: 0.01
+    c_psaci: 0.05
+    ccn_l: 300.0
+    ccn_o: 100.0
+    const_vg: false
+    const_vi: false
+    const_vr: false
+    const_vs: false
+    de_ice: false
+    do_qa: true
+    do_sedi_heat: false
+    dw_land: 0.16
+    dw_ocean: 0.1
+    fast_sat_adj: true
+    fix_negative: true
+    icloud_f: 1
+    mono_prof: true
+    mp_time: 450.0
+    prog_ccn: false
+    qi0_crt: 8.0e-05
+    qi_lim: 1.0
+    ql_gen: 0.001
+    ql_mlt: 0.001
+    qs0_crt: 0.001
+    rad_graupel: true
+    rad_rain: true
+    rad_snow: true
+    rh_inc: 0.3
+    rh_inr: 0.3
+    rh_ins: 0.3
+    rthresh: 1.0e-05
+    sedi_transport: false
+    tau_g2v: 900.0
+    tau_i2s: 1000.0
+    tau_l2v:
+    - 225.0
+    tau_v2l: 150.0
+    use_ccn: true
+    use_ppm: false
+    vg_max: 12.0
+    vi_max: 1.0
+    vr_max: 12.0
+    vs_max: 2.0
+    z_slope_ice: true
+    z_slope_liq: true
+  gfs_physics_nml:
+    cal_pre: false
+    cdmbgwd:
+    - 3.5
+    - 0.25
+    cnvcld: false
+    cnvgwd: true
+    debug: false
+    dspheat: true
+    fhcyc: 24.0
+    fhlwr: 3600.0
+    fhswr: 3600.0
+    fhzero: 0.25
+    hybedmf: true
+    iaer: 111
+    ialb: 1
+    ico2: 2
+    iems: 1
+    imfdeepcnv: 2
+    imfshalcnv: 2
+    imp_physics: 11
+    isol: 2
+    isot: 1
+    isubc_lw: 2
+    isubc_sw: 2
+    ivegsrc: 1
+    ldiag3d: true
+    lwhtr: true
+    ncld: 5
+    nst_anl: true
+    pdfcld: false
+    pre_rad: false
+    prslrd0: 0.0
+    random_clds: false
+    redrag: true
+    shal_cnv: true
+    swhtr: true
+    trans_trac: true
+    use_analysis_sst: true
+    use_ufo: true
+  interpolator_nml:
+    interp_method: conserve_great_circle
+  nam_stochy:
+    lat_s: 96
+    lon_s: 192
+    ntrunc: 94
+  namsfc:
+    fabsl: 0
+    fabss: 0
+    faisl: 99999
+    faiss: 99999
```

In reviewing things a bit more, I think these relate to the land / sea / sea-ice mask, so we should set them to zero too.  I want to double check though by doing a short test simulation.
```suggestion
    faisl: 0  # Use land / sea / sea-ice mask from forcing files instead of initial conditions
    faiss: 0  # Use land / sea / sea-ice mask from forcing files instead of initial conditions
```

---

I committed this but LMK if you changed your mind here

---

Thanks, yeah, I still think this change is good.

---

(deleted my previous comment)

Actually I do want to double check how the sea / land / sea-ice mask is generated online.  In looking at the forcing files it doesn't appear that there is a climatology for that.  I see a static ocean/land mask (with no sea-ice) in the form of `seaice_newland.grb` (despite the name of the file).  I'll probably need to look carefully at the fortran code.

---

The sea ice fraction climatology clearly comes from `CFSR.SEAICE.1982.2012.monthly.clim.grb`.

---

So setting these to 0 is OK then?

---

See [notebook](https://github.com/ai2cm/explore/blob/master/brianh/2022-05-24-land-sea-mask-namelist/2022-05-24-land-sea-mask-debugging.ipynb) showing that if these parameter are set to 0, the land sea mask is dynamically calculated from the sea ice fraction, and that if they are set to 99999, it is persisted. Not sure what happens if `faiss` and `faisl` are set to one and `fsicl` and `fsics` are set to another, probably something to avoid. I think this confirms that we want all of these to be set to 0 in the PIRE and nudge-to-25km contexts, and 99999 in the nudge to 40-day SHiELD context.

---

Thanks for doing those runs and confirming that @brianhenn!  I'm happy with this now if you'd to merge.